todos/app/resources/lists/resource.rb

```diff
@@ -4,10 +4,25 @@ module Lists
   class Resource < ApplicationResource
     filter_by :project, optional: true
 
+    sort_by :sorting
```

Comment:
@danielweinmann I didn't see how to use a block here, since we need to apply it to a scope. I did make it work by creating a model method.

Reply:
I started working on passing a block to sort_by and using `define_singleton_method` inside Model, but the API wasn't much simpler, it required me to create a method name anyway, and in my opinion, the Collection code got more complicated.